### PR TITLE
Update the debugging lesson

### PR DESCRIPTION
**Pull request by vlisovsk**
Here, the updated script from the previous lesson is used (from #103).
All the output logs updated.

### CODE REVIEW

File: second-analysis-steps/fixing-errors.md [hunk 1]
```diff
@@ -129,89 +129,138 @@ And then run it:
...
+Before               INFO | | +--> /ODIN [No Address]  LHCb::ODIN
+Before               INFO   +--> /Phys [No Address]  DataObject
+Before               INFO   | +--> /StdAllLooseKaons [No Address]  DataObject
+Before               INFO   | | +--> /Particles [No Address]  KeyedContainer<LHCb::Particle,Co [0x9]
+Before               INFO   | | +--> /decayVertices [No Address]  KeyedContainer<LHCb::Vertex,Cont [0]
+Before               INFO   | | +--> /Particle2VertexRelations [No Address]  LHCb::Relation1D<LHCb::Particle,
+Before               INFO   | | +--> /_RefitPVs [No Address]  KeyedContainer<LHCb::RecVertex,C [0]
+Before               INFO   | +--> /Sel_D0 [No Address]  DataObject
+Before               INFO   | | +--> /Particles [No Address]  KeyedContainer<LHCb::Particle,Co [0]
+Before               INFO     +--> /TupleDstToD0pi_D0ToKpi [No Address]  DataObject
```
Comment (chrisburr) [inline second-analysis-steps/fixing-errors.md]: Should we manually edit this to be `TupleDstToD0pi_D0ToKK`?

File: second-analysis-steps/fixing-errors.md [hunk 2]
```diff
@@ -129,89 +129,138 @@ And then run it:
...
 After                INFO   | +--> /StdAllLooseKaons [No Address]  DataObject
-After                INFO   | | +--> /Particles [No Address]  KeyedContainer<LHCb::Particle,Co [7]
+After                INFO   | | +--> /Particles [No Address]  KeyedContainer<LHCb::Particle,Co [0x9]
 After                INFO   | | +--> /decayVertices [No Address]  KeyedContainer<LHCb::Vertex,Cont [0]
 After                INFO   | | +--> /Particle2VertexRelations [No Address]  LHCb::Relation1D<LHCb::Particle,
 After                INFO   | | +--> /_RefitPVs [No Address]  KeyedContainer<LHCb::RecVertex,C [0]
-After                INFO     +--> /Sel_D0 [No Address]  DataObject
+After                INFO   | +--> /Sel_D0 [No Address]  DataObject
+After                INFO   | | +--> /Particles [No Address]  KeyedContainer<LHCb::Particle,Co [0]
+After                INFO     +--> /TupleDstToD0pi_D0ToKpi [No Address]  DataObject
```
Comment (chrisburr) [inline second-analysis-steps/fixing-errors.md]: [As above](https://github.com/lhcb/starterkit-lessons/pull/106/files#r186638781)

File: second-analysis-steps/fixing-errors.md (hunk 1 above)
Comment (vlisovsk) [reply second-analysis-steps/fixing-errors.md]: Well spotted, thanks!

**Comment by vlisovsk** [comment]
Actually, now there's another problem in the challenge formatting: since we are talking about D*, it considers * as the _italic_ formatting

**Pull request merged**